rice-middleware/impl/src/main/java/org/kuali/rice/kew/impl/document/search/DocumentSearchCriteriaBoLookupableHelperService.java

```diff
@@ -942,13 +942,17 @@ protected void executeColumnCustomization(ResultRow resultRow, DocumentSearchRes
         populateCustomColumns(customColumns, searchResult);
 
         // if there is an action custom column, always put that before any other field
+        List<Column> columnsToRemove = new ArrayList<Column>();
```

Comment:
I agree with Claus since there will never be more than one item in the columnsToRemove list.